react-components/src/architecture/concrete/pointsOfInterest/ads/PointsOfInterestAdsProvider.ts

```diff
@@ -60,7 +60,9 @@ export class PointsOfInterestAdsProvider implements PointsOfInterestProvider<Ext
       );
     }
 
-    return result.data.items.map(poiItemToInstance);
+    return await Promise.all(
+      result.data.items.map(async (item) => await poiItemToInstance(item, this._sdk))
+    );
```

Comment:
This will potentially fire a lot of parallel requests :see_no_evil: Can we maybe create a map like we do with the PoI comments? Later on, I'd want a shared cache for these names and the commentors' names